cisco_secure_endpoint/CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# CHANGELOG - cisco_secure_endpoint
+
+<!-- towncrier release notes start -->
```

Comment:
I'm confused by this changelog entry.

Is this a logs integration or does it only have assets?

Reply:
@iliakur these "without assets" PRs are meant to establish the integration / integration id / source type id so that the crawler can be developed for it. so we don't typically review the asset content in these PRs.

Reply:
Aha, so if there's a crawler that needs to be written, could you use some template that adds python package scaffolding as well please?

Our changelogs are of 2 kinds:
- autogenerated for any kind of python package (logs, crawlers, checks). These **need to only have the towncrier header** line at the beginning.
- manually maintained for tile-only integrations.

Reply:
Hehe, the `validate-repository` workflow is complaining about this exact thing in fact ;)

Reply:
Ah I see... @ankitarajput-crest I think you need to remove this line (and just a heads-up that if you use the `metrics_crawler` type when running `ddev create` it'll generate this file correctly for you)

Reply:
@nathanmadams will this end up having python code though? If yes, we need to **keep** the towncrier line and get rid of the 1.0.0 release entry.

Reply:
Python code won't be a part of this repo but crawler repo.

Reply:
@bhargavnariyanicrest sounds good! in that case let's do as Nathan says: we get rid of the towncrier header and keep the v1.0.0 release section.